frontend/src/components/navigation/NavLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useLocation } from 'react-router-dom'
+import styles from './NavLink.module.css'
+
+interface LinkProps {
+    location_path: string
+    linkName: string
+}
+
+function NavLink({location_path, linkName}: LinkProps) {
```

Comment:
Convention naming: [Google](https://google.github.io/styleguide/tsguide.html#identifiers)
```suggestion
function NavLink({locationPath, linkName}: LinkProps) {
```

Reply:
done: aa2fb016ce76f2ae269fa73c60f086544ecd620d